test failure looks relevant (pun intended)
```
FAIL: Suite should include sync-request if searching is configured
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/test_suite_sync_request.py", line 35, in test_sync_request
    self.assertXmlPartialEqual(self.get_xml('sync_request'), suite, "./sync-request[1]")
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 26, in assertXmlPartialEqual
    self.assertXmlEqual(expected, actual, normalize=False)
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 32, in assertXmlEqual
    _check_shared(expected, actual, LXMLOutputChecker(), "xml")
  File "/mnt/commcare-hq-ro/corehq/apps/app_manager/tests/util.py", line 126, in _check_shared
    raise AssertionError(message)
AssertionError: XML mismatch
--- want.xml
+++ got.xml
@@ -1,6 +1,6 @@
 <partial>
   <sync-request>
-    <post relevant="&#10;count(instance('casedb')/casedb/case[@case_id=instance('querysession')/session/data/case_id]) = 0&#10;" url="https://www.example.com/a/test_domain/phone/claim-case/">
+    <post relevant="count(instance('casedb')/casedb/case[@case_id=instance('querysession')/session/data/case_id]) = 0" url="https://www.example.com/a/test_domain/phone/claim-case/">
       <data key="case_id" ref="instance('querysession')/session/data/case_id"/>
     </post>
     <command id="search_command.m0">
```

--- a/corehq/apps/app_manager/const.py
+++ b/corehq/apps/app_manager/const.py
@@ -54,6 +54,4 @@ AMPLIFIES_NOT_SET = 'not_set'
 
 DEFAULT_MONTH_FILTER_PERIOD_LENGTH = 0
 
-CLAIM_DEFAULT_RELEVANT_CONDITION = """
-count(instance('casedb')/casedb/case[@case_id=instance('querysession')/session/data/case_id]) = 0
-"""
+CLAIM_DEFAULT_RELEVANT_CONDITION = "count(instance('casedb')/casedb/case[@case_id=instance('querysession')/session/data/case_id]) = 0"

